### ISSUE TITLE
Bug with displaying PNG in version 8.2.1

### ISSUE THREAD

**Issue by sergey-yabloncev** [mpdf/mpdf#1950]
### Guidelines

- [X] I understand that [if I fail to adhere to contribution guidelines and/or fail to provide all required details, this issue may be closed without review](https://github.com/mpdf/mpdf/blob/development/.github/CONTRIBUTING.md).

### Description of the bug

When updating mpdf from version 8.2.0 to version 8.2.1, an issue appeared with displaying barcodes in the PNG/base64 format. 
Instead of the image, a simple black square is displayed.

### mPDF version

8.2.1

### PHP Version and environment (server type, cli provider etc., enclosing libraries and their respective versions)

8.2

### Reproducible PHP+CSS+HTML snippet suffering by the error

<?php

require_once __DIR__ . '/vendor/autoload.php';

$img = '<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHsAAAAiAQMAAABFiDQDAAAABlBMVEX///8AAABVwtN+AAAAAXRSTlMAQObYZgAAAAlwSFlzAAAOxAAADsQBlSsOGwAAACFJREFUKJFjuDzTyijPP/fMttR3L+e+lk1gGBUYFSBFAABgzSo6/tjlxwAAAABJRU5ErkJggg==" />';
$mpdf = new Mpdf\Mpdf();
$mpdf->WriteHTML($img);
$mpdf->OutputFile(__DIR__ . '/file.pdf');

**Comment by sergey-yabloncev** [comment]
```php

<?php

require_once __DIR__ . '/vendor/autoload.php';

$img = '<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHsAAAAiAQMAAABFiDQDAAAABlBMVEX///8AAABVwtN+AAAAAXRSTlMAQObYZgAAAAlwSFlzAAAOxAAADsQBlSsOGwAAACFJREFUKJFjuDzTyijPP/fMttR3L+e+lk1gGBUYFSBFAABgzSo6/tjlxwAAAABJRU5ErkJggg==" />';
$mpdf = new Mpdf\Mpdf();
$mpdf->WriteHTML($img);
$mpdf->OutputFile(__DIR__ . '/file.pdf');


```

**Comment by sergey-yabloncev** [comment]
![8 2 0](https://github.com/mpdf/mpdf/assets/32197653/0399edc7-966d-499f-9bb9-bcc195b32461)
![8 2 1](https://github.com/mpdf/mpdf/assets/32197653/df37aec0-442a-4729-ac2a-37c6d8bd625d)

**Comment by finwe** [comment]
Released 8.2.2 with reverted PNG handling change, should resolve this
